notes/2022-01-26.md

```diff
@@ -0,0 +1,84 @@
+# 2022-01-26
+
+Lead Scribe: Lily
+
+## Admin
+
+- sorry about notes
+- private github repo --> Spring 2022
+- grading contract FYI 
+    - Will be given further instructions on ways to achieve an 'A' or 'B'
+        - To get a 'B' you will only need to complete the paper and presentation
+        - To get an 'A' you will implement a project (translation)
+    - Paper and presentation will be assigned
+        - Paper --> CS Conference Style
+        - Draft due: last day before the presentation, will be posted for the class to review
+
+
+## Opening Question 
+
+What kind of data are you most in working with?
+
+- Class response:
+    - GIS data
+    - Linguistic data (tweets, reddit posts)
+    - Numerical data (tabular)
+    - Video/Image
+    - Time series 
+    - EHR/Medical related data
+    - NLP
+    - tabular/survey
+    
+
+## How to Read a Paper
+
+
+## Model Based ML
+- Discrete probabilities (distributions introduced in murder mystery chapter)
+- Bernoulli
+- Priors (probablistic guess about a random variable)
+    - Are useful for working with less data to create strong inferences
+        - Working with things when not a lot of data is available
+    - Assumptions, expressed in a probability distribution
+- Posterior
+    - Inference given regularizer: Likelihood...
+    - Most common posterior probability distribution we're doing: Probability of parameters given data
+- Point Estimate
+    - This are the single values produced after training (weights)
+    - Posterior mean
+![very separable data](/images/00-eytrmV6b6ZOCwq9UpSYeI9oiiuA3-7daeae0f-b595-4ce3-bdd1-dee4e59d1ccd.svg)
```

Comment:
the part in the ` []` should be alt text (a description of the image)

Reply:
I've updated the description of the images, but I'm not sure how in depth I was supposed to be. If I should be more or less descriptive let me know :)

Reply:
that's good enought